lib/internal/freeze_intrinsics.js

```diff
@@ -24,14 +24,18 @@
 'use strict';
 
 module.exports = function() {
+  const {
+    Symbol,
+  } = primordials;
+
```

Comment:
Sorry I haven't seen it before. This file shouldn't be changed (The ESLint rule is disabled in here).

Reply:
Oh, i didn't have see the comment for this rule ... 
Sorry 🙏 i have updated and commit this.

Did you want a rebase to push this commit in my last commit ?

Reply:
Thanks. Whoever lands the PR will rebase.